### PR TITLE
BIND forwarders fix + DNSSEC validation. Issue #10507

### PR DESCRIPTION
**Pull request by vktg**
- [X] Redmine Issue: https://redmine.pfsense.org/issues/10507
- [X] Ready for review

Add the 'zone "." { }' statement only if recursion is yes AND there are no forwarders configured under the Settings tab.

It also adds `dnssec-validation` option, as many DNS forwarders don't work without this option:
```
Jul  5 17:48:44 pf4 named[11684]: no valid RRSIG resolving 'ru/DS/IN': 77.88.8.8#53
Jul  5 17:48:44 pf4 named[11684]: view zone1: validating ru/DS: no valid signature found
Jul  5 17:48:45 pf4 named[11684]: broken trust chain resolving 'mail.ru/A/IN': 77.88.8.8#53
```

### CODE REVIEW

File: dns/pfSense-pkg-bind/files/usr/local/pkg/bind.xml [hunk 1]
```diff
@@ -290,6 +290,12 @@
 			<type>checkbox</type>
 			<enablefields>bind_forwarder_ips</enablefields>
 		</field>
+		<field>
+			<fielddescr>DNSSEC Validation</fielddescr>
+			<fieldname>bind_dnssec_validation</fieldname>
+			<description>Enable recursive BIND server to perform DNSSEC validation.</description>
```
Comment (jim-p) [inline dns/pfSense-pkg-bind/files/usr/local/pkg/bind.xml:296]: Seems a little weirdly worded.

Maybe this: "Enable DNSSEC validation when acting as a recursive BIND server"
Or this: "Enable DNSSEC validation when BIND is acting as a recursive resolver"

Comment (vktg) [reply dns/pfSense-pkg-bind/files/usr/local/pkg/bind.xml]: Ok, fixed

**Comment by rbgarga** [comment]
@vktg please rebase your fork and fix conflicts

**Comment by vktg** [comment]
Ok, rebased

**Comment by rbgarga** [comment]
@vktg can you please squash the commits?

**Comment by vktg** [comment]
Done, squashed

**Pull request merged**